docs/faq.md

```diff
@@ -79,23 +79,31 @@ good to them.
 
 Probably yes. We run private experiments for this purpose.
 Please reach out to us at fuzzbench@google.com. If we agree to benchmark your
-fuzzer, please follow the guide on
-[adding a new fuzzer]({{ site.baseurl }}/getting-started/adding-a-new-fuzzer/)
-on how to integrate your fuzzer with FuzzBench.
+fuzzer:
 
-You can ignore the sections on [Requesting an experiment]({{ site.baseurl }}/getting-started/adding-a-new-fuzzer/#requesting-an-experiment) and
+1. Follow the guide on
+[adding a new fuzzer]({{ site.baseurl }}/getting-started/adding-a-new-fuzzer/)
+on how to integrate your fuzzer with FuzzBench. You can ignore the sections on
+[Requesting an experiment]({{ site.baseurl }}/getting-started/adding-a-new-fuzzer/#requesting-an-experiment) and
 [Submitting your integration]({{ site.baseurl }}/getting-started/adding-a-new-fuzzer/#submitting-your-integration).
-Please test your fuzzer works with our benchmarks, we don't have CI to verify
-this for private experiments.
-Ideally, you should test all benchmarks using `make -j test-run-$FUZZER-all`.
-This takes too long on most machines so you should at least test a few of them:
+
+2. Run `make presubmit` to validate the fuzzer's name and integration code.
+
+2. Test your fuzzer works with our benchmarks as we don't have CI to verify
```

Comment:
nit: i know it doesn't matter in md, but might change to 3.

Reply:
yes, right, i actually can just use 1., most people do that rather than doing ordering themselves.